testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -489,11 +527,13 @@ def test_seeking(self, u, reference_positions):
 
         assert_almost_equal(u.dimensions, reference_positions['box'][1],
                             decimal=5)
-        assert_almost_equal(u.atoms[0].position,
-                            reference_positions['atom1_pos'][1],
+        pos = (reference_positions['atom1_pos'][1] - 
+            reference_positions['mins'][1])
+        assert_almost_equal(u.atoms[0].position, pos,
```

Comment:
As above, if you're doing a line change, might as well switch to `assert_allcose`.